runtimes/eden/Cargo.toml

```diff
@@ -122,12 +126,19 @@ try-runtime = [
   "pallet-utility/try-runtime",
   "pallet-aura/try-runtime",
   "pallet-collator-selection/try-runtime",
+  "pallet-contracts/try-runtime",
   "cumulus-pallet-aura-ext/try-runtime",
   "cumulus-pallet-dmp-queue/try-runtime",
   "cumulus-pallet-parachain-system/try-runtime",
   "cumulus-pallet-xcm/try-runtime",
   "parachain-info/try-runtime",
 ]
+# Make contract callable functions marked as __unstable__ available. Do not enable
+# on live chains as those are subject to change.
+contracts-unstable-interface = [
+  "pallet-contracts/unstable-interface"
+]
+
```

Comment:
Soooo shall we or shall we not enable this?

Reply:
We would need to enable this only for local tests

Reply:
Can this be set to tests only?

Reply:
It's not for unit tests, but this feature should only be enabled manually while building the runtime. So as long as it's not appearing in our build instructions which is the case now, it's not doing anything.

Reply:
I see, got it